# pyo3 0.14.2 (as locked in Cargo.lock) does not support Python 3.10 and 3.11
# error: the configured Python interpreter version (3.1) is lower than PyO3's minimum supported version (3.6)
# 
# This patch updates to pyo3 0.17.*
# At time of writing pyo3 0.17.3 is used
# 
--- tokenserver-auth/Cargo.toml.orig	2023-11-30 02:34:56.000000000 +0000
+++ tokenserver-auth/Cargo.toml	2023-11-30 18:24:35.953670206 +0000
@@ -14,7 +14,7 @@
 
 async-trait = "0.1.40"
 dyn-clone = "1.0.4"
-pyo3 = { version = "0.14", features = ["auto-initialize"] }
+pyo3 = { version = "0.17", features = ["auto-initialize"] }
 reqwest = { version = "0.10.10", features = ["json", "rustls-tls"] }
 syncserver-common = { path = "../syncserver-common" }
 tokenserver-common = { path = "../tokenserver-common" }
